packages/kbn-monaco/xjson/README.md

```diff
@@ -0,0 +1,40 @@
+# README
+
+This folder contains the language definitions for XJSON used by the Monaco editor.
+
+## Summary of contents
+
+Note: All source code.
+
+### ./worker
+
+The worker proxy and worker instantiation code used in both the main thread and the worker thread.
+
+### ./dist
+
+The transpiled, production-ready version of the worker code that will be loaded by Monaco client side.
+Currently this is not served by Kibana but raw-loaded as a string and served with the source code via
+the "raw-loader".
+
+See the related ./webpack.xjson-worker.config.js file that is runnable with Kibana's webpack with:
+
```

Comment:
We don't need to commit this anymore right? We can just build the worker in the build script for the `@kbn/monaco` package and pull in the built version from `target`?